### PR TITLE
Add more ibc assets

### PR DESCRIPTION
**Pull request by samsiegart**
fixes https://github.com/Agoric/wallet-app/issues/153
fixes https://github.com/Agoric/wallet-app/issues/138

Tested deposit/withdraw with each token on mainnet

## Screenshot

image

### CODE REVIEW

**Comment by samsiegart** [comment]
cloudflare failing to build due to memory limit, looking into it...
```
11:47:15.389 | Cloning repository...
-- | --
11:47:16.182 | From https://github.com/Agoric/wallet-app
11:47:16.182 | * branch            2a8b13dd36f52dc7fb5413ebaa9c574d4eaf2bca -> FETCH_HEAD
11:47:16.182 |  
11:47:16.224 | HEAD is now at 2a8b13d fix: change explorer from bigdipper to nodesguru
11:47:16.225 |  
11:47:16.310 |  
11:47:16.311 | Using v2 root directory strategy
11:47:16.334 | Success: Finished cloning repository files
11:55:56.969 | Failed: build exceeded memory limit and was terminated
```

**Comment by samsiegart** [comment]
> cloudflare failing to build due to memory limit, looking into it...
> 
> ```
> 11:47:15.389 | Cloning repository...
> -- | --
> 11:47:16.182 | From https://github.com/Agoric/wallet-app
> 11:47:16.182 | * branch            2a8b13dd36f52dc7fb5413ebaa9c574d4eaf2bca -> FETCH_HEAD
> 11:47:16.182 |  
> 11:47:16.224 | HEAD is now at 2a8b13d fix: change explorer from bigdipper to nodesguru
> 11:47:16.225 |  
> 11:47:16.310 |  
> 11:47:16.311 | Using v2 root directory strategy
> 11:47:16.334 | Success: Finished cloning repository files
> 11:55:56.969 | Failed: build exceeded memory limit and was terminated
> ```

Simply retrying the build worked

File: wallet/src/util/ibc-assets.ts [hunk 1]
```diff
@@ -64,4 +64,67 @@ export const ibcAssets: IbcAssets = {
...
+    },
+    deposit: {
+      sourcePort: 'transfer',
+      sourceChannel: 'channel-148',
+      denom: 'stuosmo',
+    },
+    withdraw: {
+      sourcePort: 'transfer',
+      sourceChannel: 'channel-59',
+      denom:
```
Comment (turadg) [inline wallet/src/util/ibc-assets.ts:84]: it's hard to review whether all these values are correct. I end up looking at other repos. Let's just source from them https://github.com/Agoric/wallet-app/issues/155

**Review by turadg — APPROVED** [review]

File: wallet/src/util/ibc-assets.ts (hunk 1 above)
Comment (samsiegart) [reply wallet/src/util/ibc-assets.ts:84]: Were you able to find this denom in chain registry? I ended up just looking at a raw keplr txn to figure it out

Comment (turadg) [reply wallet/src/util/ibc-assets.ts:84]: https://github.com/cosmology-tech/chain-registry/blob/14e096249ae904190d83615f569521e9ec02e118/packages/assets/src/mainnet/agoric.ts#L715-L725

but the ibc denoms don't match.

**Pull request merged**